src/styles/style.scss

```diff
@@ -0,0 +1,128 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+:root {
+  --min-margin-side: 16px;
+  --color-button: #00acdc;
+}
+
+.card {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  &__logo {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__description {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+    margin: 0 var(--min-margin-side) 4px;
+  }
+
+  &__code {
+    text-align: left;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+    margin-bottom: 16px;
+    margin-left: var(--min-margin-side);
+  }
+
+  &__stars {
+    display: flex;
+    justify-content: space-between;
+    margin: 0  var(--min-margin-side) 24px;
+  }
+
+  &__prices {
+    display: flex;
+    justify-content: space-between;
+    margin: 0 var(--min-margin-side) 16px;
+    box-sizing: border-box;
+  }
+
+  &__text {
+    color: #616070;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__amount {
+    color: #060b35;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  &__link-buy {
+    border-radius: 5px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-align: center;
+    text-transform: uppercase;
+    color: #fff;
+    text-decoration: none;
+    padding: 12px 70px 12px 69px ;
+    box-sizing: border-box;
+    display: block;
+    margin: 0 var(--min-margin-side) 16px;
+  }
+
+  &:hover {
+    background: #fff;
+    border-color: var(--color-button);
+    color: var(--color-button);
+  }
+}
+
+.stars__container {
+  display: flex;
+  width: 96px;
+
+  &__reviews-text {
+    font-weight: 400;
+    font-size: 10px;
+    color: #060b35;
+    text-align: center;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &star {
```

Comment:
This syntax means using `.stars__containerstar` selector. I this there is an error.